crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -640,15 +640,29 @@ pub struct RoomInfo {
     pub(crate) notification_counts: UnreadNotificationsCount,
     /// The summary of this room.
     pub(crate) summary: RoomSummary,
-    /// Flag remembering if the room members are synced.
-    pub(crate) members_synced: bool,
     /// The prev batch of this room we received during the last sync.
     pub(crate) last_prev_batch: Option<String>,
+    /// How much we know about this room.
+    pub(crate) sync_info: SyncInfo, // FIXME: Serialization change!
```

Comment:
@poljar I haven't written a store migration before. Adding one for this would (1) be required and (2) not too hard, right?

Reply:
Yes I think so. I think can just deserialize into a default value of `SyncInfo::IncompleteMembers`. This will refetch the members for some rooms, but that's alright.

Reply:
Oh, so we don't even need a migration that reads the old field and writes the new. Interesting idea!

Reply:
Yeah, I think it should be fine to just ignore the old field in this case.